src/app/Http/Requests/FinancialOperations/EditOperationRequest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Http\Requests\FinancialOperations;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rules\File;
+
+class EditOperationRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
```

Comment:
An update, i. e. a PUT request, should expect the same data as a POST request, with all the fields that are required in a POST being required in a PUT as well. If you really want this behavior, then use PATCH instead.

Reply:
Ok, it makes more sense to rework the request then. Is it alright to omit "account_id" in update request, as the update doesn't use it (it's not really tied to the resource itself)?